src/test/java/cosminsechel1/tests/tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,93 @@
+package cosminsechel1.tests.tema6;
```

Comment:
Put tema6 package inside cosminsechel.tests package. No nee to create cosminsechel1 package. Thanks!